### PR TITLE
Feature/add GitHub views

### PR DESCRIPTION
**Pull request by michallaskowski**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [x] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [x] The project does not generate any warnings or errors
- [x] The tests are passing
- [x] Branch name is in line with [GitFlow](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/)

### CODE REVIEW

File: GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//  GitHubTableDataSource.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 21/05/16.
+//
+
+import UIKit
+
+
+class GitHubTableDataSource: NSObject, UITableViewDataSource {
```
Comment (krris) [inline GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift]: Why here is `NSObject`?

File: GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private let cellID = "cell"
+    var items: [GitHubCellViewModel] = []
+
+    func setUpTableView(tableView: UITableView) {
+        tableView.registerNib(UINib(nibName: "GitHubCell", bundle: nil), forCellReuseIdentifier: cellID)
+        tableView.dataSource = self
+    }
+
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return min(3, items.count)
```
Comment (krris) [inline GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift]: I would move `3` to a constant variable.

File: GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        countLabelText = "\(prCount)"
+        nameLabelText = repo.name.uppercaseString
+        color = UIColor.colorForPRCount(prCount)
+    }
+}
+
+private extension UIColor {
+    class func colorForPRCount(count: Int) -> UIColor {
+        switch count {
+        case 0 ..< 4: return UIColor(red: 246/255, green: 157/255, blue: 67/255, alpha: 1)
```
Comment (krris) [inline GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift]: Use `gcb_fadedOrangeColor`. We have UIColor extension in GCBCore.

File: GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+final class GitHubCell: UITableViewCell {
+
+    @IBOutlet private weak var countLabel: UILabel!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var circleView: EllipseView!
+
+    func configureWithViewModel(viewModel: GitHubCellViewModel) {
+        countLabel.textColor = viewModel.color
```
Comment (krris) [inline GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift]: WDYT about removing `color` from view model? Instead I would add a state of a cell and based on the state choose a color in GitHubCell.

File: GrandCentralBoard/Widgets/GitHub/GitHubCell.xib [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                        <rect key="frame" x="32" y="50" width="42" height="21"/>
+                        <fontDescription key="fontDescription" type="system" pointSize="40"/>
+                        <color key="textColor" red="0.70980392160000005" green="0.023529411760000001" blue="0.14901960780000001" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                        <nil key="highlightedColor"/>
+                    </label>
+                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="REPO NAME" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Rga-c2-hq0">
+                        <rect key="frame" x="138" y="40" width="494" height="41"/>
+                        <fontDescription key="fontDescription" type="system" pointSize="34"/>
```
Comment (krris) [inline GrandCentralBoard/Widgets/GitHub/GitHubCell.xib]: Fix font letter spacing.

**Comment by krris** [comment]
I finished CR @michallaskowski

File: GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift (hunk 1 above)
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift]: The protocol needs it
`public protocol UITableViewDataSource : NSObjectProtocol`

File: GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift (hunk 3 above)
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift]: Yes, but the red is different than one in GCBCore. I am not sure if it should change, when `gcb_fadedOrangeColor` in core is changed. But I will change it for now.

File: GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift (hunk 4 above)
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/GitHub/View/GitHubCell.swift]: I think view model should hold final values for the view. That means strings, colors, UIImages etc. - not values that are used to compute the final color.

**Comment by krris** [comment]
Good job 👏

**Pull request merged**